rotkehlchen/db/filtering.py

```diff
@@ -974,6 +975,8 @@ def make(
             )
         if exclude_ignored_assets is True:
             filters.append(DBIgnoredAssetsFilter(and_op=True, asset_key='asset'))
+        if identifier is not None:
```

Comment:
Do you really need to use the filter mechanism here? This is not filterable and it's just a `SELECT` that should match a single entry. No need to complicate things. Do a SELECT query directly and don't add more `ifs` and `args` in here.

The identifier has no place in the filtering mechanism as it's not something that can be combined with anything else.